sql/core/src/test/scala/org/apache/spark/sql/TPCSchema.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql
+
+/**
+ * Base trait for TPC related tests.
+ *
+ * https://www.tpc.org/tpc_documents_current_versions/current_specifications5.asp
+ */
+trait TPCSchema {
```

Comment:
this trait doesn't seem to provide any value. Is there a place that matches `TPCSchema` and call its methods?

Reply:
The fields in this trait were required in both TPCHSchema as well as in TPCDSSchema. So, created this new trait to keep the info at single place. Both TPCHSchema and TPCDSSchema extends from this trait.